### ISSUE TITLE
Add accessibility sections to all components that react to keyboard events

### ISSUE THREAD

**Issue by michalkvasnicak** [Byteclaw/visage#468]
Change titles to `Keyboard Interaction`

TODO:

- [x] - `Accordion`
- [x] - `Banner`
- [x] - `Checkbox`
- [x] - `Drawer`
- [x] - `Message`
- [x] - `Radio`
- [x] - `Toggle`
- [x] - `Tooltip`

**Comment by michalkvasnicak** [comment]
I'm not sure if the section should be called `Accessibility` or `Keyboard shortcuts`. But I'm leaning towards `Keyboard shortcuts`.

@jurajhrib what's your opinition? WAI aria uses `Keyboard shortcuts`. Shopify uses the same https://polaris.shopify.com/components/actions/button#navigation
